lib/tapioca/compilers/dsl/active_record_columns.rb

```diff
@@ -347,6 +347,25 @@ def do_not_generate_strong_types?(constant)
           Object.const_defined?(:StrongTypeGeneration) &&
               !(constant.singleton_class < Object.const_get(:StrongTypeGeneration))
         end
+
+        def handle_unknown_type(column_type)
+          return "T.untyped" unless column_type.ancestors.include?(ActiveModel::Type::Value)
```

Comment:
Can we change this to be:
```suggestion
          return "T.untyped" unless column_type < ActiveModel::Type::Value
```